app/database/MetricsDB.scala

```diff
@@ -56,5 +57,58 @@ class MetricsDB(implicit val db: Database) {
       }
 
   def getDistinctNewspaperBooks: Either[ProductionMetricsError, Seq[Option[String]]] =
-    await(db.run(metricsTable.filter(book => !book.newspaperBook.isEmpty && book.newspaperBook =!= "").map(_.newspaperBook).distinct.result))
+    await(db.run(metricsTable.filter(!_.newspaperBook.isEmpty).map(_.newspaperBook).distinct.result))
```

Comment:
Should include `book.newspaperBook =!= ""` in the filter as we want to filter out empty strings

Reply:
Yes, git went a bit mad when I rebased this, sorry